.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,29 @@
+name: Pull Request
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: '${{ secrets.PAT_DECRAPIFIER }}'
```

Comment:
We only need this if we're going to run duti stuff. You should be able to do:
```
    - name: Checkout
      uses: actions/checkout@v2
```

A little extra background on `PAT_DECRAPIFIER` it's only needed since duti pushes up commits which you can't do with the default github token in the secretes.

Reply:
done.